python/fusion_engine_client/analysis/data_loader.py

```diff
@@ -135,7 +135,7 @@ class DataLoader(object):
 
     logger = logging.getLogger('point_one.fusion_engine.analysis.data_loader')
 
-    def __init__(self, path=None, save_index=True, ignore_index=False):
+    def __init__(self, path=None, save_index=True, ignore_index=False, source_ids: List[int]=[0]):
```

Comment:
This should be optional and should default to `None`. Do not assume that 0 is present, or is the only ID that the user wants by default.

Reply:
Correction: remove this altogether

Reply:
The problem that I'm seeing with getting rid of this is that the mixed log reader needs the source IDs in its constructor, which isn't called until the data loader's `open` function.